conformance/reports/v1.0.0/cilium/v1.15.0-pre.3-report.yaml

```diff
@@ -38,7 +38,7 @@ profiles:
         - HTTPRouteRequestTimeout
         - HTTPRouteResponseHeaderModification
         - HTTPRouteSchemeRedirect
-      name: HTTP
+    name: HTTP
```

Comment:
If  an implementation adds a broken-format report, will it mess up the display completely, or just for that implmentation?

Reply:
good question - it will be missing from the display. For example, currently the `generate_table` filters for the name "HTTP", and if that column is missing (the format results in the column being in an incorrect location in the dataframe), then it will be dropped. I guess it's somewhat of a check